### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per @mriccell request

### CODE REVIEW

File: documentation/staging/content/userguide/platforms/environments.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Container Services for use with Kubernetes* on OCI Compute, and on "Authorized C
 
 [Oracle Linux Cloud Native Environment](https://docs.oracle.com/en/operating-systems/olcne/) is a fully integrated suite for the development and management of cloud-native applications. Based on Open Container Initiative (OCI) and Cloud Native Computing Foundation (CNCF) standards, Oracle Linux Cloud Native Environment delivers a simplified framework for installations, updates, upgrades, and configuration of key features for orchestrating microservices.
 
-WebLogic Server and the WebLogic Kubernetes Operator are certified and supported on Oracle Linux Cloud Native Environment. Operator 2.6.0 provides certified support of OLCNE 1.1 with Kubernetes 1.17.0.
+WebLogic Server and the WebLogic Kubernetes Operator are certified and supported on Oracle Linux Cloud Native Environment. Operator 3.2.5 provides certified support of OLCNE 1.3 with Kubernetes 1.2.0 and CRI-O 1.20.2.
```
Comment (rjeberhard) [inline documentation/staging/content/userguide/platforms/environments.md:26]: 1.2.0 -> 1.20.2 (Kubernetes and CRI-O versions match).

I'm remembering more of the conversation and I think that we should preserve the statement that operator 2.6.0 is certified on OLCNE 1.1 and that 3.2.5 is certified on OLCNE 1.3.

File: documentation/staging/content/userguide/prerequisites/introduction.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ weight: 2
 
 For the current production release 3.2.5:
 
-* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+ (check with `kubectl version`).
+* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+, 1.2.0+  (check with `kubectl version`).
```
Comment (rjeberhard) [inline documentation/staging/content/userguide/prerequisites/introduction.md:10]: 1.20.2+ :)

File: documentation/staging/content/userguide/platforms/environments.md (hunk 1 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/platforms/environments.md:26]: Thank you. Fixed.

File: documentation/staging/content/userguide/prerequisites/introduction.md (hunk 2 above)
Comment (rosemarymarano) [reply documentation/staging/content/userguide/prerequisites/introduction.md:10]: (Duh) Thank you. Fixed.

File: documentation/staging/content/userguide/prerequisites/introduction.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ weight: 2
 
 For the current production release 3.2.5:
 
-* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+ (check with `kubectl version`).
+* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+, 1.20.6+  (check with `kubectl version`).
```
Comment (rjeberhard) [inline documentation/staging/content/userguide/prerequisites/introduction.md:10]: The "and" should move

File: documentation/staging/content/userguide/prerequisites/introduction.md [hunk 4]
```diff
@@ -7,10 +7,10 @@ weight: 2
...
 * Flannel networking v0.9.1-amd64 or later (check with `docker images | grep flannel`), Calico networking v3.16.1 or later,
  *or* OpenShift SDN on OpenShift 4.3 systems.
-* Docker 18.9.1 or 19.03.1+ (check with `docker version`) *or* CRI-O 1.14.7+ (check with `crictl version | grep RuntimeVersion`).
+* Docker 18.9.1 or 19.03.1+ (check with `docker version`) *or* CRI-O 1.20.2+ (check with `crictl version | grep RuntimeVersion`).
```
Comment (rjeberhard) [inline documentation/staging/content/userguide/prerequisites/introduction.md:13]: This is also 1.20.6

Comment (rosemarymarano) [reply documentation/staging/content/userguide/prerequisites/introduction.md]: Done.

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**